hack/setup-kind-csi-cluster.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -x
+cd $GOPATH/src/github.com/libopenstorage/openstorage
+if [ $DELETE_CLUSTER ]; then
+	kind delete cluster --name kind-csi 
+fi
+kind create cluster --name kind-csi --config hack/kind.yaml --image kindest/node:v1.17.0
+kind_worker_id=$(docker inspect --format="{{.Id}}" kind-csi-control-plane)
+docker cp $GOPATH/bin/osd ${kind_worker_id}:/osd
+docker exec -ti ${kind_worker_id} bash -c "mkdir -p /var/lib/kubelet/plugins/osd.openstorage.org"
+docker exec -tid ${kind_worker_id} bash -c "pid=$(ps -ef | grep osd | grep -v grep | awk '{print $2}'); kill -9 $pid"
+docker exec -tid ${kind_worker_id} bash -c "CSI_ENDPOINT=/var/lib/kubelet/plugins/osd.openstorage.org/csi.sock /osd -d --driver=name=fake --sdkport 9106 --sdkrestport 9116"
```

Comment:
here pass in the csi name as `osd.openstorage.org`

Reply:
Fixed